packages/fxa-settings/src/components/CardHeader/index.stories.tsx

```diff
@@ -49,3 +59,35 @@ export const WithCustomServiceName = storyWithProps({
   headingWithCustomServiceFtlId: MOCK_CUSTOM_HEADING_FTL_ID,
   headingText: MOCK_HEADING,
 });
+
+export const WithSeparateSubheadingDefaultServiceName = storyWithProps(
+  {
+    headingText: MOCK_HEADING,
+    headingTextFtlId: MOCK_DEFAULT_HEADING_FTL_ID,
+    subheadingWithDefaultServiceFtlId: MOCK_DEFAULT_HEADING_FTL_ID,
+    subheadingWithLogoFtlId: MOCK_DEFAULT_HEADING_FTL_ID,
+  },
+  'Separate l10n for subheading, with default service'
+);
+
+export const WithSeparateSubheadingCustomServiceName = storyWithProps(
+  {
+    serviceName: MOCK_SERVICE_NAME,
+    headingText: MOCK_HEADING,
+    headingTextFtlId: MOCK_DEFAULT_HEADING_FTL_ID,
+    subheadingWithCustomServiceFtlId: MOCK_CUSTOM_HEADING_FTL_ID,
+    subheadingWithLogoFtlId: MOCK_CUSTOM_HEADING_FTL_ID,
+  },
+  'Separate l10n for subheading, with custom service name'
+);
+
+export const WithSeparateSubheadingLogo = storyWithProps(
```

Comment:
I think this one makes more sense to feel more generic in case any other service logos have images later - we're just showing here that displaying an image is an option.